src/components/Global/InfiniteDatePager.tsx

```diff
@@ -1,24 +1,31 @@
 import React, { useRef, useCallback, useEffect } from "react";
 import { View, Text, StyleSheet } from "react-native";
 import InfinitePager from "react-native-infinite-pager";
+import {InfinitePagerImperativeApi, InfinitePagerPageComponent} from "react-native-infinite-pager/src";
```

Comment:
Résoud 1 des 2 erreurs du node_modules
```suggestion
import {InfinitePagerImperativeApi, InfinitePagerPageComponent} from "react-native-infinite-pager";
```

Reply:
Chez moi ça avait rien changé mais si ça fonctionne super 😃